### DISCUSSION TITLE
Create web site on github

### DISCUSSION THREAD

**Discussion by psychic-journey-ec961076** [community/community#102394] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi everyone! Hope you are very well!

How can I create domain for my website like this? With .pages.github.io subdomain
![image](https://github.com/community/community/assets/142434856/8029bf8d-d09f-40d7-9bf0-068a5b35b7ad)

**Comment by fury-05** [comment]
You can create repository then add you website to that repository, then go to setting then click on pages on you left column
![image](https://github.com/community/community/assets/103818328/d9b2ace1-230e-4847-a798-9eed4ae4953b)
Then select the branch from which you want to deploy your website
![image](https://github.com/community/community/assets/103818328/3a888d6c-0002-4e65-b76a-e82bd0901f6b)
Click on save and that's it you will have you website up can running on GitHub Pages 

Note: Remember to make repository public if you are not on Pro Plan of GitHub or else you will not be able to use GitHub Pages to publish your website.

**Reply by mr-h4ck3r-x** [reply]
it's helpful!

**Reply by psychic-journey-ec961076** [reply]
Yes, but when I do this, it creates the page name https://thangteasas.github.io. I want to create website name https://thangteasas.pages.github.io/

**Comment by surendratiwari3** [comment, marked as answer]
@psychic-journey-ec961076 

when you create pages with normal github account
https://thangteasas.github.io/

when you create pages with  github enterprise account
https://thangteasas.pages.github.io/

**Reference:**
https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site
